qcelemental/physical_constants/ureg.py

```diff
@@ -75,6 +75,8 @@ def build_units_registry(context):
     for k, v in phys_const_map.items():
         ureg.define(f"{k} = {phys_const[v]['value']} * {phys_const[v]['unit']}")
 
+    ureg.define(f"au_pressure = au_energy / au_length**3")  # not in CODATA
```

Comment:
I suspect it is the fact that this can be derived from other units. CODATA tends not to duplicate.

Reply:
codata isn't averse to duplicating but guess they skipped this one.